Automattic/wp-calypso
# Gravatar: Add a `flow` parameter into the `fetchMagicLoginAuthenticate` action to facilitate the customization of 2FA SMS messages

Related to 105880-Automattic/gravatar

## Proposed Changes

* Adding a `flow` parameter into the `fetchMagicLoginAuthenticate` for the Gravatar-powered clients

## Testing Instructions



* Setup Calypso by referring to [this doc](https://github.com/Automattic/wp-calypso#getting-started)
* Check out this PR
* Run `yarn install` and then `yarn start` to run Calypso web app
* See this Phabricator diff: D123426-code

## Pre-merge Checklist
